private/aws-client-retry-test/src/ClientRetryTest.spec.ts

```diff
@@ -18,11 +18,12 @@ describe("Middleware-retry integration tests", () => {
       body: Readable.from(""),
     }),
   };
-  const getInsightCommand = new GetInsightCommand({
-    InsightId: "foo",
+  const getInsightCommand = new HeadObjectCommand({
```

Comment:
Also needs to be updated on lines 32, 49 and so on.
I can't provide suggestion as those files are not changed.

```suggestion
  const headObjectCommand = new HeadObjectCommand({
```